packages/ckeditor5-utils/tests/manual/duplicated-import/duplicated-import.html

```diff
@@ -2,6 +2,10 @@
 	<meta name="x-cke-crawler-ignore-patterns" content='{
 		"uncaught-exception": "CKEditorError: ckeditor-duplicated-modules"
 	}'>
+
+	<meta http-equiv="Content-Security-Policy"
+		content="connect-src 'self' https://cksource.com http://*.cke-cs.com https://cdn.ckeditor.com">
+	<script src="https://cdn.ckeditor.com/ckeditor5/44.2.0/ckeditor5.umd.js" crossorigin></script>
```

Comment:
Editor version is hardcoded in this CDN link, but I'm not sure if this is something we should be concerned.